src/Bundle/Cli/Dumper/EmailDetailsDumper.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\MailerBundle\Cli\Dumper;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Twig\Loader\LoaderInterface;
+
+final class EmailDetailsDumper implements EmailDetailDumperInterface
+{
+    public function __construct(
+        private readonly array $emails,
+        private readonly ?TranslatorInterface $translator,
+        private readonly LoaderInterface $templateLoader,
+    ) {
+    }
+
+    public function dump(InputInterface $input, OutputInterface $output, string $code): void
+    {
+        $email = $this->emails[$code];
+
+        $subject = $email['subject'] ?? '';
+
+        if ($this->translator !== null) {
+            $subject = $this->translator->trans($subject);
```

Comment:
What language will be used as fallback in cli env? Should we specify `%sylius.locale%` over here or it will happen automatically? For example if `%sylius.locale%` is french, will it output in french or english?

Reply:
Same for `EmailsListDumper.php`

Reply:
it will be translated according to `_locale`, so config of the app